JASP-Engine/JASP/R/MixedModelsCommon.R

```diff
@@ -676,79 +676,70 @@
-  } else{
-    is_REML <- lme4::isREML(model$full_model)
-  }
   
-  fitStats$addColumnInfo(name = "deviance",
-                         title = gettext("Deviance"),
-                         type = "number")
-  if (is_REML) {
-    fitStats$addColumnInfo(
-      name = "devianceREML",
-      title = gettext("Deviance (REML)"),
-      type = "number"
-    )
-  }
-
+  ### fit statistics
+  fitStats <- createJaspTable(title = gettext("Fit statistics"))
+  fitStats$position <- 1
+  
+  fitStats$addColumnInfo(name = "deviance", title = gettext("Deviance"), type = "number")
+  if (lme4::isREML(full_model))
+    fitStats$addColumnInfo(name = "devianceREML", title = gettext("Deviance (REML)"), type = "number")
   fitStats$addColumnInfo(name = "loglik", title = gettext("log Lik."), type = "number")
   fitStats$addColumnInfo(name = "df",     title = gettext("df"),       type = "integer")
   fitStats$addColumnInfo(name = "aic",    title = gettext("AIC"),      type = "number")
   fitStats$addColumnInfo(name = "bic",    title = gettext("BIC"),      type = "number")
+  jaspResults[["fitSummary"]][["fitStats"]] <- fitStats
   
-  jaspResults[["fitStats"]] <- fitStats
   
+  temp_row <- list(
+    deviance = deviance(full_model, REML = FALSE),
+    loglik   = logLik(full_model),
+    df       = attr(logLik(full_model) , "df"),
+    aic      = AIC(full_model),
+    bic      = BIC(full_model)
+  )
   
-  if (is.list(model$full_model)) {
-
-    temp_row <- list(
-      deviance = deviance(model$full_model[[length(model$full_model)]], REML = FALSE),
-      loglik   = logLik(model$full_model[[length(model$full_model)]]),
-      df       = attr(logLik(model$full_model[[length(model$full_model)]]) , "df"),
-      aic      = AIC(model$full_model[[length(model$full_model)]]),
-      bic      = BIC(model$full_model[[length(model$full_model)]])
-    )
-  
-    if (is_REML)
-      temp_row$devianceREML <- lme4::REMLcrit(model$full_model[[length(model$full_model)]])
-    
-  }else{
-
-    temp_row <- list(
-      deviance = deviance(model$full_model, REML = FALSE),
-      loglik   = logLik(model$full_model),
-      df       = attr(logLik(model$full_model) , "df"),
-      aic      = AIC(model$full_model),
-      bic      = BIC(model$full_model)
-    )
-    
-    if (is_REML)
-      temp_row$devianceREML <- lme4::REMLcrit(model$full_model)
-    
-  }
+  if (lme4::isREML(full_model))
+    temp_row$devianceREML <- lme4::REMLcrit(full_model)
   
   fitStats$addRows(temp_row)
-  fitStats$addFootnote(.mmMessageFitType(is_REML))
+  fitStats$addFootnote(.mmMessageFitType(lme4::isREML(full_model)))
+  
+  
+  ### sample sizes
+  fitSizes <- createJaspTable(title = gettext("Sample sizes"))
+  fitSizes$position <- 2
+  
+  fitSizes$addColumnInfo(name = "observations", title = gettext("Observations"), type = "integer")
+  temp_row <- list(
+    observations = nrow(full_model@frame)
+  )
+  for (n in names(full_model@flist)) {
```

Comment:
I'd recommend using something more descriptive than `n` here. I personally use `neem`

Reply:
I'm not sure that `neem` is more descriptive than `n`. I, personally, had to google it up. Accordingly, I could use `jmeno` instead.

Reply:
neem is the Dutch pronunciation of "name", but you can use `thisName` for instance. A one letter reference is always unclear.

Reply:
changed